instrumentation/events/lttng-module/net.h

```diff
@@ -148,6 +312,31 @@ LTTNG_TRACEPOINT_EVENT_CLASS(net_dev_template,
 		ctf_integer_hex(void *, skbaddr, skb)
 		ctf_integer(unsigned int, len, skb->len)
 		ctf_string(name, skb->dev->name)
+		ctf_enum(net_network_header, unsigned char, network_header_type, __get_network_header_type(skb))
```

Comment:
So is it that the physical layer has the information about the network layer protocol, and we don't want to trace the physical layer information ?